crates/tests/databases-tests/src/postgres/snapshots/databases_tests__postgres__cli_version3_tests__postgres_current_only_configure_initial_configuration_is_unchanged.snap

```diff
@@ -883,6 +883,31 @@ expression: default_configuration
         "foreignRelations": {},
         "description": null
       },
+      "custom_test_cidr": {
```

Comment:
It's a bit weird that we also update v3 and v4 test snapshots for this change, but I suppose it follows from each version using sharing the same test database definition.